### DISCUSSION TITLE
dump_binary_path

### DISCUSSION THREAD

**Discussion by potenzatk** [spatie/laravel-backup#1117] [Q&A]
'dump' => [
      'dump_binary_path' => 'laravel7\storage', // only the path, so without 'mysqldump' or 'pg_dump'.
      'use_single_transaction', // perform dump using a single transaction.
      'timeout' => 60 * 10, // 10 minute timeout
]

We can pass any folder path here dump_binary_path or we need to provide specific path here if I used wamp server.

 When I run php artisan backup:run I got below error. can you please help me. also I found so may pepel face this issue. so please help here it will help other to thanks.
Backup failed because The dump process failed with exitcode 1 : General error : '"laravel\storage/mysqldump"' is not recognized as an internal or external command,
operable program or batch file.

Thanks

**Comment by pathros** [comment]
In this case, `dump_binary_path` refers to the path of the mysqldump executable program.

Try setting `/usr/bin/` instead of `laravel7\storage`.

[Take a look at my question](https://github.com/spatie/laravel-backup/discussions/1124) for more details.
